test/inductor/test_fp8.py

```diff
@@ -231,9 +238,11 @@ def fp8_saturated(x, dtype):
     @unittest.skipIf(not SM90OrLater, "FP8 is only supported on H100+")
     @parametrize(
         "float8_dtype",
-        (torch.float8_e4m3fn, torch.float8_e5m2)
-        if torch.version.hip is None
-        else (torch.float8_e4m3fnuz, torch.float8_e5m2fnuz),
+        (
+            (torch.float8_e4m3fn, torch.float8_e5m2)
+            if torch.version.hip is None
+            else (torch.float8_e4m3fnuz, torch.float8_e5m2fnuz)
+        ),
```

Comment:
@dnikolaev-amd Do we still need this or can we use `_fix_fp8_dtype_for_hip` function here?

Reply:
fixed